app/user/serializers.py

```diff
@@ -1,116 +1,72 @@
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils.http import urlsafe_base64_decode
 from rest_framework import serializers
-from rest_framework.validators import UniqueValidator
 
-from app.user.token import account_activation_token
-from app.user.validators import (
-    validate_password_digit,
-    validate_password_lowercase,
-    validate_password_symbol,
-    validate_password_uppercase,
-)
-from speaksfer.settings.base import EMAIL_USER
+from app.user.utils import create_reset_email, generate_reset_token, send_email
 
 User = get_user_model()
 
 
-class UserSerializer(serializers.ModelSerializer):
+class PasswordResetSerializer(serializers.Serializer):
 
-    username = serializers.CharField(
-        max_length=20,
-        min_length=8,
-        validators=[UniqueValidator(queryset=User.objects.all())],
-    )
+    email = serializers.EmailField(required=True)
 
-    email = serializers.EmailField(
-        required=True,
-        validators=[UniqueValidator(queryset=User.objects.all())],
-    )
+    class Meta:
+        fields = ["email"]
 
-    password = serializers.CharField(
-        max_length=128,
-        min_length=8,
-        write_only=True,
-        validators=[
-            validate_password_digit,
-            validate_password_uppercase,
-            validate_password_symbol,
-            validate_password_lowercase,
-        ],
-    )
+    def validate(self, attrs: Any) -> Any:
 
-    class Meta:
-        model = User
-        fields = ("email", "username", "password")
-
-    @staticmethod
-    def send_email(user: Any, request: Any) -> None:
-
-        current_site_info = get_current_site(request)
-        email_body = render_to_string(
-            "email_verification.html",
-            {
-                "user": user,
-                "domain": current_site_info.domain,
-                "uid": urlsafe_base64_encode(force_bytes(user.pk)),
-                "token": account_activation_token.make_token(user),
-            },
-        )
-
-        send_mail(
-            "Verify  your email!",
-            email_body,
-            EMAIL_USER,
-            [user.email],
-            fail_silently=False,
-        )
-
-    def create(self, validated_data: Any) -> Any:
         request = self.context.get("request")
-        user = User.objects.create_user(**validated_data)
-        self.send_email(user, request)
+        email = attrs.get("email")
+        user = generate_reset_token(email)
+
+        if user:
+            email_data = create_reset_email(request, *user)
+            send_email("password_reset.html", email_data)
 
-        return user
+        return super().validate(attrs)
 
 
-class VerifyEmailSerializer(serializers.Serializer):
-    token = serializers.CharField()
-    uidb64 = serializers.CharField()
+class VerifyPasswordResetSerializer(serializers.Serializer):
+
+    password = serializers.CharField(
```

Comment:
Add password validators like during registering here so that the passwoed is strong

Reply:
Okay